### PR TITLE
Standardize page title in templates to include bureau name

### PR DESCRIPTION
**Pull request by anselmbradford**
Numerous pages did not include `| Consumer Financial Protection Bureau` or included it in another format. This adds it to our template's page titles where it was absent or wrong.

## Changes

- Add title suffix `| Consumer Financial Protection Bureau` to page templates that were missing it or had it formatted wrong.

## How to test this PR

1. Pages such as https://localhost:8000/consumer-tools/financial-well-being/results/ should include the bureau name in the page title versus it being absent on production https://www.consumerfinance.gov/consumer-tools/financial-well-being/results/

### CODE REVIEW

**Comment by Scotchester** [comment]
Any of these that can display in Spanish need to include the translation tag around the name of the Bureau, à la: https://github.com/cfpb/consumerfinance.gov/blob/9693858b0df65da32501765cac538edc30c3c654/cfgov/jinja2/v1/_layouts/base.html#L57

**Comment by anselmbradford** [comment]
> Any of these that can display in Spanish need to include the translation tag around the name of the Bureau, à la:

I only added it for the pages that had translated titles already. Do we want to do that for the others where it would be a mix of English and Spanish in the title potentially? Any in particular jumping out?

**Comment by Scotchester** [comment]
Ah, sorry, didn't scroll all the way down and see that you had done some. The one that I thought needed it was `cfgov/jinja2/v1/ask-cfpb/answer-page.html`, but I confess I didn't check to confirm whether that template also serves Spanish answers or not.

**Comment by anselmbradford** [comment]
> Ah, sorry, didn't scroll all the way down and see that you had done some. The one that I thought needed it was cfgov/jinja2/v1/ask-cfpb/answer-page.html, but I confess I didn't check to confirm whether that template also serves Spanish answers or not.

Ahh yes, good catch, it is needed for that one. Fixed in https://github.com/cfpb/consumerfinance.gov/pull/6339/commits/29cd4633d49b6f180ab459029b3b93bb94fc434b

File: cfgov/agreements/jinja2/agreements/index.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {% extends "agreements/base_agreements.html" %}
 
-{% block title %}Credit card agreement database{% endblock %}
+{% block title %}
+    Credit card agreement database | Consumer Financial Protection Bureau
+{% endblock %}
```
Comment (Scotchester) [inline cfgov/agreements/jinja2/agreements/index.html:5]: This is pretty trivial, but could we collapse these new newlines wherever they appear in this PR, for consistency (see, e.g., `cfgov/diversity_inclusion/jinja2/diversity_inclusion/form-submitted.html` a couple files down in this diff already had them)

```suggestion
{% block title -%}
    Credit card agreement database | Consumer Financial Protection Bureau
{%- endblock %}
```

**Review by Scotchester — APPROVED** [review]
Approved pending one trivial consistency request.

File: cfgov/agreements/jinja2/agreements/index.html (hunk 1 above)
Comment (anselmbradford) [reply cfgov/agreements/jinja2/agreements/index.html]: Done! https://github.com/cfpb/consumerfinance.gov/pull/6339/commits/7358f3030aa64a945a16baa6127f58011eec8680

**Pull request merged**